tests/foreman/ui/test_ansible.py

```diff
@@ -862,3 +863,64 @@ def test_positive_schedule_recurring_hostgroup_job(self):
 
         :expectedresults: Scheduled Job appears in the Job Invocation list at the appointed time
         """
+
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    @pytest.mark.parametrize('setting_update', ['ansible_verbosity'], indirect=True)
+    def test_positive_ansible_job_with_verbose_stdout(
+        self,
+        target_sat,
+        module_org,
+        module_location,
+        module_ak_with_cv,
+        setting_update,
+        registered_hosts,
+    ):
+        """Verify debug mode output for host
+
+        :id: 06e0a95c-530b-11ef-a28c-36dcd2c0c415
+
+        :steps:
+            1. Select two or more hosts
+            2. Run ansible roles on those hosts
+            3. Check the host execution logs
+
+        :expectedresults: The host execution log should show the debug log for expected host instead of the all host present in the inventory.
+
+        :BZ: 1912941
+
+        :Verifies: SAT-12267
+
+        :customerscenario: true
+        """
+        if is_open('SAT-27757'):
+            setting_update.value = True
+        else:
+            setting_update.value = 'Level 3 (-vvv)'
+            setting_update.update({'value'})
```

Comment:
Also, currently, default is set as Disabled(0), so we don't need to override it, but we can just handle it with `if not is_open()`
```suggestion
        if not is_open('SAT-27757'):
            setting_update.value = '3'
            setting_update.update({'value'})
```

Reply:
This is more relevant, so I have updated the code according to this.